python/bindings/openravepy_global.cpp

```diff
@@ -248,6 +248,27 @@ class PyTriMesh
     PyTriMesh(object vertices, object indices) : vertices(vertices), indices(indices) {
     }
     PyTriMesh(const TriMesh& mesh) {
+#ifdef USE_PYBIND11_PYTHON_BINDINGS
+        // vertices
+        const int nvertices = mesh.vertices.size();
+        const int numel = nvertices * 3;
+        std::vector<dReal> vvertices(numel);
```

Comment:
@tgn3000 is there a way to do this without creating a new `vector`?

Reply:
yes, done; same for the rest